modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+The new `iframe_template_callback` option, provided by https://github.com/Namstel[Namstel], is an addition to the media plugin.
+
+The `iframe_template_callback` option takes a function and replaces the default callback when the source URL matches a valid iframe.
+
+It works equivalently to the `audio_template_callback` and `video_template_callback` options, but for iframes.
+
+
+=== New `dispatchChange` UndoManager API to fire the change with current editor status as `level` and current undoManager layer as `lastLevel`
+
+The new `dispatchChange` API allows for the generating of an undo level outside the current editor’s context.
+
+It takes the current editor undo level status as `level` and the current `undoManager` layer as `lastLevel`. When a change event fires, however, `dispatchChange` does not put the change in undo level on to the `undoManager` stack.
+
+// An example would be a good thing. Also for the updated docs proper.
+
+
+
+[[known-issues]]
+== Known issues
+
+This section describes issues that users of {productname} 6.1 may encounter and possible workarounds for these issues.
+
+There are three known issues affecting {productname} 6.1.
+
+=== Clearer focus states for buttons while navigating with keyboard does not work in older browsers
+
+{productname} 6.1 introduced a new usability feature: a blue outline that more clearly highlights the currently active button when navigating {productname} dialogs with a keyboard.
+
+This feature relies on the `:focus-visible` pseudo-selector.
+
+This pseudo-selector is supported in current and recent versions of all major browsers, but it is not supported in older browser versions.
+
+Specifically, for `:focus-visible` to work, the following browser versions, or later, are required:
+
+[%header,cols="1,>1"]
+|===
+|Browser
+|Oldest supported version
+
+|Google Chrome
+|86
+
+|Microsoft Edge
+|86
+
+|Mozilla Firefox
+|85
+
+|Opera
+|72
+
+|Safari
+|15.4
+|=== 
+
+
+=== If a `codesample` is the first element added to a {productname} document, the selection is incorrectly set to inside the `contenteditable="false"` block
+
+When adding a `codesample` (by, for example, choosing *Insert > Code sample…*), the added material is placed within a `contenteditable="false"` block.
+
+This block can be selected and copied but not edited. And, once copied, pasting it back adds a further `contenteditable="false"` block containing the added material, to the document.
+
+In {productname} 6.1, however, if a `codesample` is the first material added to a document, {productname} incorrectly sets the selection to the added material within the block, and not to the block itself.
+
+If this material is then copied and pasted, it is added to the document as editable text. This is not the expected behavior.
+
+There is a workaround: adding a blank line to an otherwise empty {productname} 6.1 document before adding a `codesample` results in the selection being sets to the added block, as expected.
```

Comment:
This isn't the known issue that had to be documented... the known issue was that pasting a codesample over top of another codesample resulted in incorrect behaviour. The selection issue when it was the first element was a side case found and has been there for quite some time, so it's not something we should be mentioning here.

- [ ] Fix known issue